libcodechecker/cmd/cmd_line_client.py

```diff
@@ -468,71 +389,10 @@ def bug_hash_filter(bug_id, filepath):
                                            else "")
 
 
-def handle_auth_requests(args):
-    session = session_manager.SessionManager_Client()
-
-    auth_token = session.getToken(args.host, args.port)
-
-    auth_client = authentication_helper.ThriftAuthHelper(args.host,
-                                                         args.port,
-                                                         '/Authentication',
-                                                         auth_token)
-    try:
-        handshake = auth_client.getAuthParameters()
-
-        if not handshake.requiresAuthentication:
-            LOG.info("This server does not require privileged access.")
-            return
-
-        if auth_token and handshake.sessionStillActive:
-            LOG.info("You are already logged in.")
-            return
-        else:
-            LOG.info("Server requires authentication to access. Please use "
-                     "'CodeChecker cmd login' to authenticate.")
-
-    except TApplicationException:
-        LOG.info("This server does not support privileged access.")
-        return
+def handle_login(args):
 
+    login = True
     if 'logout' in args:
-        logout_done = auth_client.destroySession()
-        if logout_done:
-            session.saveToken(args.host, args.port, None, True)
-            LOG.info("Successfully logged out.")
-        return
-
-    methods = auth_client.getAcceptedAuthMethods()
-    # Attempt username-password auth first.
-    if 'Username:Password' in str(methods):
-        username = args.username
-
-        # Try to use a previously saved credential from configuration file.
-        saved_auth = session.getAuthString(args.host, args.port)
-
-        if saved_auth:
-            LOG.info("Logging in using preconfigured credentials...")
-            username = saved_auth.split(":")[0]
-            pwd = saved_auth.split(":")[1]
-        else:
-            LOG.info("Logging in using credentials from command line...")
-            pwd = getpass.getpass("Please provide password for user '{0}'"
-                                  .format(username))
-
-        LOG.debug("Trying to login as {0} to {1}:{2}"
-                  .format(username, args.host, args.port))
-        try:
-            session_token = auth_client.performLogin("Username:Password",
-                                                     username + ":" +
-                                                     pwd)
-
-            session.saveToken(args.host, args.port, session_token)
-            LOG.info("Server reported successful authentication.")
-        except shared.ttypes.RequestFailed as reqfail:
-            LOG.error("Authentication failed! Please check your credentials.")
-            LOG.error(reqfail.message)
-            sys.exit(1)
-    else:
-        LOG.critical("No authentication methods were reported by the server "
-                     "that this client could support.")
-        sys.exit(1)
+        login = False
```

Comment:
Statement can be simplified: `login = 'logout' in args`. And then the variable can be inlined into the function call.